schema/errors.go

```diff
@@ -173,3 +173,43 @@ func NewNoVerificationStrategyResponsible() error {
 		Messages: new(text.Messages).Add(text.NewErrorValidationVerificationNoStrategyFound()),
 	})
 }
+
+func NewHookValidationError(instancePtr, message string, messages text.Messages) *ValidationError {
+	return &ValidationError{
+		ValidationError: &jsonschema.ValidationError{
+			Message:     message,
+			InstancePtr: instancePtr,
+		},
+		Messages: messages,
+	}
+}
+
+type ValidationListError struct {
+	Validations []*ValidationError
+}
+
+func (e ValidationListError) Error() string {
+	return fmt.Sprintf("%d validation errors occurred", len(e.Validations))
```

Comment:
Shouldn't we also be able to see the concrete errors?

Reply:
Fix in 75a55945